triton_backend/all_models/multimodal/multimodal_encoders/1/model.py

```diff
@@ -530,17 +662,67 @@ def postprocess_output_tensors(
                 prompt_vocab_size_tensor = pb_utils.Tensor(
                     'OUT_PROMPT_VOCAB_SIZE', prompt_vocab_size.astype(np.int32))
 
-                output_tensors = [
+                response_tensors = [
                     prompt_embedding_table_tensor, prompt_vocab_size_tensor
                 ]
 
                 inference_response = pb_utils.InferenceResponse(
-                    output_tensors=output_tensors)
+                    output_tensors=response_tensors)
                 responses.append(inference_response)
         # You should return a list of pb_utils.InferenceResponse. Length
         # of this list must match the length of `requests` list.
         return responses
 
+    def run_vision_encoder(self, vit_input: Dict[str,
+                                                 torch.Tensor]) -> torch.Tensor:
+        batch_size = [v.shape[0] for v in vit_input.values()]
+        assert all(
+            b == batch_size[0]
+            for b in batch_size), "Batch sizes of encoder inputs must match"
+        batch_size = batch_size[0]
+
+        embeddings = []
+        for start_idx in range(0, batch_size, self.vision_max_batch_size):
+            end_idx = min(start_idx + self.vision_max_batch_size, batch_size)
+            logger.info(
```

Comment:
```suggestion
            logger.verbose(
```

To avoid print statements for every request.